scripts/moose_test_runner

```diff
@@ -26,6 +26,13 @@ if not os.path.isdir(MOOSE_PYTHON_DIR):
     sys.exit(1)
 
 sys.path.append(MOOSE_PYTHON_DIR)
+MOOSE_HIT_DIR = os.path.join(MOOSE_DIR, 'moosetools', 'contrib', 'hit')
+if not os.path.isdir(MOOSE_HIT_DIR):
+    MOOSE_HIT_DIR = os.path.join(MOOSE_DIR, 'framework', 'contrib', 'hit')
+if not os.path.isdir(MOOSE_HIT_DIR):
+    print('Could not locate the HIT directory', file=sys.stderr)
+    sys.exit(1)
```

Comment:
This isn't quite right.  Where do you plan on installing moose tools into?  That's the directory we'll have to look in.  In the installed binary case - MOOSE_DIR in this script becomes PREFIX.  So I presume this will need to be something like `$MOOSE_DIR/share/moose/moosetools`.  Then we'll need to update moose.mk to copy/install the moosetools files to that location/dir.

Reply:
Can we just install hit.so at the same location as pyhit? If so, we can just take the changes I made to the __init__ file and forget this change.

Reply:
But where is pyhit installed to - that's what I meant.  `hit.so` is copied/installed regardless of where it lives (as long as we link it via applibs var in the makefile).  It's the hit python bindings that will need to be addressed since you are moving their src location (I think).

Reply:
I took this out.